src/components/badge/index.tsx

```diff
@@ -143,6 +147,15 @@ class Badge extends PureComponent<BadgeProps> {
       height: badgeHeight,
       minWidth: badgeHeight
     };
+    if (withCounterIcon) {
```

Comment:
I think it's more accurate to check if the user has passed both icon and label together, no?

Reply:
Yup